README-offloaded-indexing.md

```diff
@@ -0,0 +1,79 @@
+### Cluster with remote indexing node, this a little manual since deploy vars are getting messy
+    $ export GIT_REL='v103rc2' && export ENCD_REL='v103rc2' && echo "$GIT_REL $ENCD_REL"
+
+    # Normal elasticsearch cluster
+    $ bin/deploy -b "$GIT_REL" --cluster-name "$ENCD_REL" --es-wait --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Head frontend node: 12 app workers, 8 index workers
+    $ export ES_IP=172.31.25.24
+    $ bin/deploy -b $GIT_REL --es-ip $ES_IP --cluster-name "$ENCD_REL" --remote-indexing --primary-indexing --index-procs 8 --app-workers 12 --pg-open --do-batchupgrade no --dry-run
```

Comment:
I think --primary-indexing is redundant here?   It appears to be on by default, unless you specify --no-indexing?
Also --do-batchupgrade no is set by --release-candidate or --candidate, right?

Reply:
I've been mentioning this in various configuration management discussions.   The answer is unfortunately no. 
 The dependencies between arguments in the config are not straightforward. 

We really need to clean up the configuration management before getting too much further into our scaling endeavors.